### PR TITLE
docs(webhook): Inventory webhook description.

### PR DESCRIPTION
**Pull request by merlin-northern**
Changelog: Title
Ticket: MEN-7450

### CODE REVIEW

**Comment by merlin-northern** [comment]
@estenberg this is more fyi Eystein, we will have to adjust the existing screenshots later.

**Review by MuchoLucho — APPROVED** [review]
I have not tested the steps but it looks reasonable to me

**Review by tranchitella — APPROVED** [review]

File: 08.Server-integration/07.Webhooks/docs.md [hunk 1]
```diff
@@ -4,9 +4,14 @@ taxonomy:
     category: docs
 ---
 
-Mender supports Webhooks to send data to third-party systems. With this mechanism, it's possible to notify external applications about device lifecycle events. This helps you avoid polling and manual synchronization between Mender and other systems.
+Mender supports Webhooks to send data to third-party systems. With this mechanism, it's possible
+to notify external applications about device lifecycle and inventory changed events. This helps
+you avoid polling and manual synchronization between Mender and other systems, as well as allows
+you to handle device-related events and integrate Mender into your ecosystem seamlessly.
```
Comment (nickanderson) [inline 08.Server-integration/07.Webhooks/docs.md:10]:
```suggestion
to notify external applications about device lifecycle and inventory changed events. This eliminates the need for manual polling and synchronization, allowing for efficient handling of device-related events.
```

File: 08.Server-integration/07.Webhooks/docs.md [hunk 2]
```diff
@@ -4,9 +4,14 @@ taxonomy:
...
 
-This integration is available in all Mender plans, as well as Mender Open Source.
+We divided webhook integrations into two categories (_scopes_): `deviceauth` and `inventory`.
+The former is available in all Mender plans, as well as Mender Open Source,
```
Comment (nickanderson) [inline 08.Server-integration/07.Webhooks/docs.md:13]: I want to suggest a link to mender plans here when mentioned. I dunno how stable that url is. https://mender.io/pricing/plans

File: 08.Server-integration/07.Webhooks/docs.md [hunk 3]
```diff
@@ -16,21 +21,30 @@ You need a device integrated with Mender, see the [Get started guide](../../01.G
 
 ### External service
 
-You need an external service or application exposed on the internet and able to receive data from Mender in the form of an HTTP POST request.
+You need an external service or application exposed on the internet and able to receive data
+from Mender in the form of an HTTP POST request. The hostname part of the URL must resolve
+to a globally routable unicast IP address. We recommend using HTTPS and configuring
+a secret for request signing purposes (see [below](#signature-header)).
```
Comment (nickanderson) [inline 08.Server-integration/07.Webhooks/docs.md:27]:
```suggestion
To receive data from Mender, you'll need an internet-accessible service or app that can accept HTTP POST requests. The service's domain name should resolve to a publicly accessible IP address. For security, we recommend using HTTPS and setting up a secret for verifying request authenticity (see [below](#signature-header)).
```

File: 08.Server-integration/07.Webhooks/docs.md [hunk 4]
```diff
@@ -16,21 +21,30 @@ You need a device integrated with Mender, see the [Get started guide](../../01.G
...
 
-A Webhook fires on device lifecycle events:
+A webhook in the `deviceauth` scope fires on device lifecycle events:
 * device provisioning,
 * device decommissioning, or
 * its authentication status changes.
 
+### Inventory changed triggers
+
+A webhook in the `inventory` scope fires when device changed inventory events:
```
Comment (nickanderson) [inline 08.Server-integration/07.Webhooks/docs.md:40]:
```suggestion
A webhook in the `inventory` scope fires whenever Mender Server detects and updates a change in a device's inventory information.
```

File: 08.Server-integration/07.Webhooks/docs.md [hunk 5]
```diff
@@ -16,21 +21,30 @@ You need a device integrated with Mender, see the [Get started guide](../../01.G
...
+* Mender Server detected a change in the device inventory and updated it in the db
```
Comment (nickanderson) [inline 08.Server-integration/07.Webhooks/docs.md:41]:
```suggestion
```

File: 08.Server-integration/07.Webhooks/docs.md [hunk 6]
```diff
@@ -89,11 +104,125 @@ The properties included depend on the event type:
...
+```
+
+More information about payload possibilities are available
+in the [iot-manager](https://github.com/mendersoftware/iot-manager) management API specification.
 
-### Signature header
+## Signature header
 
-If you specify a secret, an integrity check is calculated and located in the `X-Men-Signature` header, which contains the HMAC-SHA256 of the payload using the configured secret.
+If you specify a secret, an integrity check is calculated and located in the `X-Men-Signature` header,
```
Comment (nickanderson) [inline 08.Server-integration/07.Webhooks/docs.md:224]: Why `X-Men`?  Could be Avengers, Fantastic Four, Justice League, Teen Titans.

**Review by nickanderson — APPROVED** [review]
Looks good to me. Take or leave suggestions for re-phrasing.

File: 08.Server-integration/07.Webhooks/docs.md (hunk 6 above)
Comment (merlin-northern) [reply 08.Server-integration/07.Webhooks/docs.md:224]: haha! a. I like XMen second: can we make it Spiderman?

File: 08.Server-integration/07.Webhooks/docs.md (hunk 1 above)
Comment (merlin-northern) [reply 08.Server-integration/07.Webhooks/docs.md]: fixed. I only could not resist and added extra sentence about integration.

File: 08.Server-integration/07.Webhooks/docs.md (hunk 2 above)
Comment (merlin-northern) [reply 08.Server-integration/07.Webhooks/docs.md]: sure.

File: 08.Server-integration/07.Webhooks/docs.md (hunk 3 above)
Comment (merlin-northern) [reply 08.Server-integration/07.Webhooks/docs.md]: fixed. I just added `unicast`. I like that word. Unicast.

File: 08.Server-integration/07.Webhooks/docs.md (hunk 5 above)
Comment (merlin-northern) [reply 08.Server-integration/07.Webhooks/docs.md]: removed.

File: 08.Server-integration/07.Webhooks/docs.md (hunk 1 above)
Comment (nickanderson) [reply 08.Server-integration/07.Webhooks/docs.md]: Most of my English teachers were not pleased with my brevity. But I hold on to the one who gave me feedback that said she liked how I would ruthlessly cut out things. I guess it's a bit of cognitive dissonance avoidance or confirmation bias. :P

**Pull request merged**